sdcm/nemesis.py

```diff
@@ -2577,7 +2557,7 @@ def disrupt(self):
         self.disrupt_stop_start_scylla_server()
 
 
-class RestartThenRepairNodeMonkey(NotSpotNemesis):
+class RestartThenRepairNodeMonkey(Nemesis):
```

Comment:
Change that back as well, or better remove the unnecessary commit.

Reply:
@roydahan I approved this removal, this code path is only relevant when this specific nemesis class is used in all other case is not being used.

People run into it only when class is used, and it's not up to date in all the logic needed.

It's only add confusion.